plugins/index.js

```diff
@@ -28,15 +42,42 @@ module.exports = (on, config) => {
       if (browser.isHeadless) {
         arguments_.args.push('--window-size=1920,1080');
       }
+      arguments_.preferences.default.profile = {
+        content_settings: {
+          exceptions: {
+            clipboard: {
+              '*': {
+                expiration: '0',
+                last_modified: '13248200230459161',
+                model: 0,
+                setting: 1,
+              },
+            },
+          },
+        },
+      };
     }
 
-    if (!process.env.SKIP_METAMASK_INSTALL) {
-      // NOTE: extensions cannot be loaded in headless Chrome
-      const metamaskPath = await helpers.prepareMetamask(
-        process.env.METAMASK_VERSION || '10.25.0',
-      );
-      arguments_.extensions.push(metamaskPath);
+    if (process.env.PROVIDERS) {
+      process.env.CYPRESS_PROVIDERS = process.env.PROVIDERS;
```

Comment:
Why do we set this here?

Reply:
cause cypress only recognizes env vars that start with CYPRESS_